### PR TITLE
Milestone3: Add and Remove

### PR DESCRIPTION
**Pull request by prolajumokeoni**
Milestone 3:
In this milestone:
- Created a module for add, remove and clear

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,9 +2,14 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
 import completeTask from './checkbox';
 import addTask from './addtodo';
+import editTodo from './edit';
+import deleteTodo from './delete';
+import clearTodo from './clearChecked';
```
Comment (Dipeshtwis) [inline src/index.js:7]: - Well Done implementing all the functions in separate `js` files. :smile:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @prolajumokeoni, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: A function for adding a new task (add a new element to the array), editing task descriptions, and deleting a task (remove an element from the array) is implemented. Well done

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] Currently, the live link pointing to the README doesn't seem professional, I am inviting you to fix it.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (prolajumokeoni) [reply src/index.js:7]: thankyou, how can i fix the link on live demo?

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Priscilla, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

<br>
Cheers..Happy coding!, and keep Soaring higher!  💻 🍷 🚀  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**